src/businesses/service/BusinessesCrudService.ts

```diff
@@ -0,0 +1,80 @@
+import Business from '../model/Business';
```

Comment:
Interesting. What was the reason of wrapping this logic. It seems like this would be better served as an implementation to an interface if you'd expect different implementations. @mb-ca

Reply:
@ryankazokas I was just following how the Orders were setup.  Do you know why it was designed that way for orders?

Reply:
Not sure @mb-ca . I was out of the loop on this when that was created. We can leave it as is but maybe @csimko can elaborate.

Reply:
Actually  i see why. The ordersActivities service uses the ordersCrud service. You can leave like this then, because we will want to add business members in the future.